services/release-controller-test/admin_02_rbac.yaml

```diff
@@ -0,0 +1,183 @@
+kind: List
+apiVersion: v1
+items:
+- kind: ServiceAccount
+  apiVersion: v1
+  metadata:
+    name: release-controller-test
+    namespace: release-controller-test
+
+# pull access to authenticated users
+- kind: RoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1beta1
+  metadata:
+    name: image-puller
+    namespace: release-controller-test-release
+  subjects:
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: system:authenticated
+  roleRef:
+    kind: ClusterRole
+    apiGroup: rbac.authorization.k8s.io
+    name: system:image-puller
+- kind: RoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1beta1
+  metadata:
+    name: user-viewer
+    namespace: release-controller-test-release
+  subjects:
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: system:authenticated
+  roleRef:
+    kind: ClusterRole
+    apiGroup: rbac.authorization.k8s.io
+    name: view
+
+# grant release-controller-test image modification in the release namespace
```

Comment:
Isn't this dangerous? Why is it necessary?

Reply:
For the purpose of our test environment, the "release" namespace is actually **release-controller-test-release**.  AFAIK the release-controller creates a "release" imagestream in the specified release namespace (release-controller-test-release) and modifies it's tags if/when the release successfully passes its suite of tests.  We can probably tighten the screws a bit here.  I'd prefer not to get too aggressive until we prove that this environment works as expected, if that's alright with you?